nixos/modules/hardware/bitcraze.nix

```
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.bitcraze;
+
+in
+{
+  options.hardware.bitcraze = {
+    enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      description = lib.mdDoc ''
+        Enables Bitcraze Crazyflie udev rules and ensures 'plugdev' group exists.
+        This is a prerequisite to using Bitcraze Crazyflie devices without being root, since Bitcraze Crazyflie USB descriptors will be owned by plugdev through udev.
+      '';
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    services.udev.packages = [ pkgs.bitcraze-udev-rules ];
+    users.groups.plugdev = { };
+  };
```

Consider adding yourself as maintainer of the module as well. That way, you'll be (or at least should be) pinged on changes.

```suggestion
  };
  
  meta.maintainers = [ vbruegge ];
```